plugins/defaultplugins/ifplugin/README.md

```diff
@@ -89,7 +89,7 @@ ifplugin <-- GOVPP : success/err
 An example of interface configuration for MEMIF in JSON format can
 be found [here](../../../cmd/vpp-agent-ctl/json/memif.json).
 
-To insert config into etcd in JSON format [vpp-agent-ctl](../../../cmd/vpp-agent-ctl/main.go)
+To insert config into etcd in JSON format [vpp-agent-ctl](../../../cmd/vpp-agent-ctl/vpp-agent-ctl.go)
```

Comment:
Maybe you could just link to the **vpp-agent-ctl** folder instead of the source file since it is split into multiple files now. This way in the future if we add some README it will automatically display that.